test/e2e/e2e.go

```diff
@@ -188,7 +65,7 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 		}
 	}
 	if framework.TestContext.NodeKiller.Enabled {
-		close(nodeKillerStopCh)
+		close(framework.NodeKillerStopCh)
```

Comment:
Overall the PR looks good except for the use `framework.NodeKillerStopCh` here. Maybe assign the NodeKillerStopCh to TestContext.NodeKiller instead? Thoughts @timothysc @spiffxp ?

Reply:
Done, looks better now. It took me a while to understand what this channel really does (cancelation), so I also added better comments to it as well.